src/DictList.elm

```diff
@@ -115,12 +130,19 @@ between an association list and a `DictList` via `toList` and `fromList`.
 # JSON
 
 @docs decodeObject, decodeArray, decodeWithKeys, decodeKeysAndValues
+
+# ListExtra
+
+@docs last, inits, (!!), uncons, maximumBy, minimumBy, andMap, andThen, takeWhile, dropWhile, unique, uniqueBy, allDifferent, allDifferentBy
+
 -}
 
-import Dict exposing (Dict)
+
+import Dict exposing (Dict, keys)
```

Comment:
It would be better not to expose `keys`, since we have our own `DictList.keys` as well. So, we can refer to `Dict.keys` explicitly.